pkgs/applications/misc/qwalkingpad/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, lib
+, qtbase
+, qtconnectivity
+, wrapQtAppsHook
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qwalkingpad";
+  version = "0.1-git.556bca7";
```

Comment:
Version `0.1` is based on their tagged Windows release. They don't tag releases for other platforms, hence the `-git.556bca7` suffix.